docs/data/sql_connectors.yml

```diff
@@ -148,7 +148,7 @@ jdbc:
     name: JDBC
     category: connector
     maven: flink-connector-jdbc
-    sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-connector-jdbc/$version/flink-connector-jdbc-$version.jar
+    sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-connector-jdbc/1.16.0/flink-connector-jdbc-1.16.0.jar
```

Comment:
Why don't we use the newly released 3.0.0 version here? :thinking:

Reply:
Good catch, that's a mistake